snapshot/deploy/docker/controller/Dockerfile

```diff
@@ -14,4 +14,9 @@
 
 FROM busybox:glibc
 COPY snapshot-controller /bin/snapshot-controller
+# Root CA certificates
+COPY /etc/pki/ca-trust /etc/pki/ca-trust
```

Comment:
this Dockerfile seems to work for me locally
```
FROM  busybox:glibc
CMD rpm -i https://www.rpmfind.net/linux/fedora/linux/development/rawhide/Everything/x86_64/os/Packages/c/ca-certificates-2017.2.20-6.fc28.noarch.rpm
CMD EXTRACT_UNSAFE_SYMLINKS=1 rpm -i ca-certificates-2017.2.20-6.fc28.noarch.rpm
```

Reply:
I'm not sure I want to rely on rpmfind.net to get the package from: I don't know what would happen to the link once the package is obsoleted. Also -- if I have rpm I can be reasonably sure I have the CA certificates on the system too.

Reply:
@tsmetana updated with fedora pkg.

Reply:
need to update /etc/ :)

Reply:
Eh... How did I... nm. Will fix this, thanks.